.cirrus.yml

```diff
@@ -234,3 +247,33 @@ freebsd_x64_stable_debug_task:
   set_toolchain_script: rustup default stable
   <<: *ELK_CARGO_FMT_AND_CLIPPY
   <<: *ELK_COMMON_BUILD_AND_TEST_DEBUG
+
+#
+# macOS
+#
+
+# Pipeline 10
+
+macos_aarch64_stable_debug_task:
+  depends_on: preflight_check
+  <<: *ELK_CONTAINER_MACOS_AARCH64
+  env:
+    PATH: /Users/admin/.cargo/bin:$PATH
+  setup_script:
+    - uname -a
+    - curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal --no-modify-path
+    - rustup component add clippy rustfmt
+  set_toolchain_script: rustup default stable
+  <<: *ELK_CARGO_FMT_AND_CLIPPY
+  <<: *ELK_COMMON_BUILD_DEBUG
+  test_script:
+    - cargo test -p elkodon_pal_concurrency_primitives
+                 -p elkodon_pal_posix
+                 -p elkodon_bb_container
+                 -p elkodon_bb_elementary
+                 -p elkodon_bb_lock_free
+                 -p elkodon_bb_log
+                 -p elkodon_bb_memory
+                 -p elkodon_bb_system_types
+                 -p elkodon_bb_testing
+                 --workspace --all
```

Comment:
This should not work since `elkodon` requires shm names longer than 30 chars, a feature that our mac platform not yet supports.